crates/ruff_workspace/src/options.rs

```diff
@@ -351,17 +351,45 @@ pub struct Options {
     pub src: Option<Vec<String>>,
 
     // Global Formatting options
-    /// The line length to use when enforcing long-lines violations (like
-    /// `E501`). Must be greater than `0` and less than or equal to `320`.
+    /// The maximum [width](http://www.unicode.org/reports/tr11/#Overview) of a line when
```

Comment:
nit: for consistency with the deprecated option documentation
```suggestion
    /// The maximum [display width](http://www.unicode.org/reports/tr11/#Overview) of a line when
```